packages/web/components/trade-tool/index.tsx

```diff
@@ -21,12 +23,37 @@ export const TradeTool: FunctionComponent<TradeToolProps> = () => {
     )
   );
 
+  // Mock
+  const unclaimedOrders = Array(0)
+    .fill(null)
+    .map((i) => i + 1);
+
   return (
     <ClientOnly>
-      <div className="relative flex flex-col gap-6 overflow-hidden md:gap-6 md:px-3 md:pb-4 md:pt-4">
+      <div className="relative flex flex-col gap-6 md:gap-6 md:px-3 md:pb-4 md:pt-4">
         <div className="flex w-full items-center justify-between">
           <SwapToolTabs activeTab={tab} setTab={setTab} />
-          {tab !== SwapToolTab.SWAP && <OrderTypeSelector />}
+          <div className="flex items-center gap-3">
+            {tab !== SwapToolTab.SWAP && <OrderTypeSelector />}
+            <Link
```

Comment:
Do we want to hide/disable this if the user's wallet isn't connected?

Reply:
Yes! Working on it

Reply:
[c1e9de5](https://github.com/osmosis-labs/osmosis-frontend/pull/3364/commits/c1e9de57396dcc80bbf0486f356175b1f28a59b0)